packages/ketcher-react/src/script/editor/tool/sgroup.ts

```diff
@@ -30,6 +30,9 @@ import LassoHelper from './helper/lasso'
 import { isEqual } from 'lodash/fp'
 import { selMerge } from './select'
 import Editor from '../Editor'
+import { startSelecting } from '../utils/selectItems/startSelecting'
+import { finishSelecting } from '../utils/selectItems/finishSelecting'
```

Comment:
updates in this file:
1) calling startSelecting and finishSelecting
2) renaming `ci` to `closestItem`
3) removing check that something is already selected